app/controllers/posts_controller.rb

```diff
@@ -42,7 +38,7 @@ def create
       format.html do
         if post_create.save
           flash[:success] = 'Post created successfully'
-          redirect_to user_posts_url
+          redirect_to user_post_url
```

Comment:
- [ ]  Kindly note that this is a wrong syntax in rails it is raising this error :point_down: 

![image](https://user-images.githubusercontent.com/65068771/162098214-391f4fed-4472-484c-bd7b-59396bc823c7.png)

User, this to redirect correctly to the new post created.

```rb
          redirect_to user_post_path(current_user_id, post_create.id)

```